docs/Quick_start_guide.rst

```diff
@@ -12,6 +12,18 @@ To install the ITK Python package::
     $ python -m pip install --upgrade pip
     $ python -m pip install itk
 
+If a pre-compiled wheel package is not found for your Python distribution, then it will
+attempt to build from source.
+
+.. note::
```

Comment:
Maybe I'm unfamiliar with the syntax but what does `.. note::` signify? For me, I typically do **Note:** <INSERT_TEXT_HERE> to draw attention to it.

Reply:
@addisonElliott This is specific to restructed text syntax. See http://docutils.sourceforge.net/docs/ref/rst/directives.html#note

It allows to generate the documentation available here: http://itkpythonpackage.readthedocs.io/en/latest/Quick_start_guide.html